app/presenters/hyrax/work_show_presenter.rb

```diff
@@ -138,6 +138,10 @@ def stats_path
       Hyrax::Engine.routes.url_helpers.stats_work_path(self)
     end
 
+    def model
+      solr_document['has_model_ssim'].first.underscore
```

Comment:
Can we use `solr_document.to_model.model_name.singular` here instead?

Reply:
👍